src/arrow_parquet/pg_to_arrow.rs

```diff
@@ -65,8 +65,8 @@ pub(crate) struct PgToArrowAttributeContext {
     is_geometry: bool,
     is_map: bool,
     attribute_contexts: Option<Vec<PgToArrowAttributeContext>>,
-    scale: Option<usize>,
```

Comment:
What are the changes here for?  Related to the `arm` support?

Reply:
not really. scale and precision is computed from Postgres `typmod`. And it is int since Postgres allows negative scales.